index.html

```diff
@@ -103,20 +103,27 @@ <h2>
         <dd>
           An [=image MIME type=].
         </dd>
+        <dt>
+          <dfn data-dfn-for="image resource">label</dfn>
+        </dt>
+        <dd>
+          A [=string=] representing the [[[#dfn-accessible-name]]] of the image.
```

Comment:
I think we want: 

```suggestion
          A [=string=] representing the [=accessible name=] of the image.
```

And then we need to get the ACCNAME spec to export that, right?

Reply:
The exporting is where I was running into issues. How do we manage that?

Reply:
I think this is it: https://github.com/w3c/respec/wiki/data-export